examples/memory/demo_memory_pinecone.py

```diff
@@ -42,5 +41,5 @@
 print("Results with filter only:", [r.content for r in results])
 
 # Clear memory
-memory.clear()
-print("Is memory empty?", memory.is_empty())
+# memory.clear()
```

Comment:
let's remove if not needed

Reply:
It was unintentional; we need it for demo purposes.